mparticle-integrations/mparticle-apple-integration-branchmetrics
# [INTENG-18507] Update to latest Branch SDK v2.1.2

## Reference
[INTENG-18507] Update to latest Branch SDK v2.1.2

## Summary
Updated the Branch SDK from v1.45.0 to v2.1.2. The latest Branch SDK version changed the import from `Branch` to `BranchSDK`, so these needed to be updated in the plugin. 

## Motivation

To provide access to the latest Branch SDK features.

## Type Of Change


## Testing Instructions

1. Check out this latest version
2. Integrate the SDK
3. Ensure all existing features are functioning









cc @BranchMetrics/saas-sdk-devs for visibility.
